docs/apache-airflow/core-concepts/backfill.rst

```diff
@@ -0,0 +1,69 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Backfill
+========
+
+Backfill is when you create runs for past dates of a dag.  Airflow provides a mechanism
+to do this through the CLI and REST API.  You provide a dag, a start date, and an end date,
```

Comment:
Should we leave a TODO for UI here? I guess we will also eventually be able to run backfill via UI?

Reply:
yeah UI portion is planned for Q4. We can update the docs once it's implemented.